pre_commit_hooks/detect_aws_credentials.py

```diff
@@ -7,62 +7,118 @@
 from six.moves import configparser
 
 
-def get_your_keys(credentials_file):
-    """reads the secret keys in your credentials file in order to be able to
-    look for them in the submitted code.
+def get_aws_credential_files_from_env():
+    """Extract credential file paths from environment variables."""
+    files = set()
+    for env_var in {'AWS_CONFIG_FILE', 'AWS_CREDENTIAL_FILE',
+                    'AWS_SHARED_CREDENTIALS_FILE', 'BOTO_CONFIG'}:
```

Comment:
p4: I'd iterate over a tuple instead of a set, you don't need set semantics here (same applies for the function below)